test/unit/tasks/data_hygiene/publishing_api_html_attachment_redirector_test.rb

```diff
@@ -0,0 +1,88 @@
+require "test_helper"
+
+class PublishingApiHtmlAttachmentRedirectorTest < ActiveSupport::TestCase
+  extend Minitest::Spec::DSL
+
+  let!(:document)               { create(:document) }
+  let!(:attachment)             { create(:html_attachment, locale: 'en') }
+  let!(:superseded_edition)     { create(:superseded_edition) }
+  let!(:edition)                {
+    create(
+      :unpublished_edition,
+                                  state: "withdrawn",
+                                  document: document,
+                                  attachments: [attachment]
+)
+  }
+  let!(:redirection)            { "www.redirection.com/attachment_path" }
```

Comment:
`www.redirection.com` is a real website. We probably want to use `www.example.com` for all our fictitious urls:

```
Example Domain
This domain is established to be used for illustrative examples in documents. You may use this domain in examples without prior coordination or asking for permission.
```

Reply:
good point, good spot
 ☝️ 👁